### ISSUE TITLE
Tutorial 2 questions

### ISSUE THREAD

**Issue by Githeo** [OpenMined/PySyft#6138]
Hello,

I'm trying to check the [Tutorial 2 - MultiLayer Split Neural Network](https://github.com/OpenMined/PySyft/blob/PySyft/syft_0.2.x/examples/tutorials/advanced/split_neural_network/Tutorial%202%20-%20MultiLayer%20Split%20Neural%20Network.ipynb)

In this tutorial, the model is split among Alice, Bob, and Claire, but it looks like Bob has neither features nor labels. So why Bob is there? 
I could have more appreciated an example where the features are split among the 3. Did you already provide such example?

Also I don't understand why, when you send the images data to the first model 

`images = images.send(models[0].location)`

the data goes to the entry part of models[0]:

```
models = [
    nn.Sequential(
                nn.Linear(input_size, hidden_sizes[0]),
                nn.ReLU(),
    ),
    nn.Sequential(
                nn.Linear(hidden_sizes[0], hidden_sizes[1]),
                nn.ReLU(),
    ),
    nn.Sequential(
                nn.Linear(hidden_sizes[1], output_size),
                nn.LogSoftmax(dim=1)
    )
]
```

Instead, when labels are sent (`labels = labels.send(models[-1].location)`), these go to the ending part of models[-1]. Infact, the shape of labels is equal to `output_size`, not `hidden_sizes[1]`.

Good job, Cheers,

**Comment by madhavajay** [comment]
0.5 is no longer supported.
